test/parallel/test-https-agent-sni.js

```diff
@@ -46,4 +50,13 @@ server.listen(0, function() {
       rejectUnauthorized: false
     }, expectResponse(j));
   }
+  https.get({
+    agent: agent,
+
```

Comment:
Is the line break intentional?

Reply:
Just following the style of the rest of the file.